[2025-03-11T07:50:56.013+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.my_task manual__2025-03-11T07:50:50+00:00 [queued]>
[2025-03-11T07:50:56.017+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.my_task manual__2025-03-11T07:50:50+00:00 [queued]>
[2025-03-11T07:50:56.017+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-11T07:50:56.024+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): my_task> on 2025-03-11 07:50:50+00:00
[2025-03-11T07:50:56.026+0000] {standard_task_runner.py:57} INFO - Started process 4427 to run task
[2025-03-11T07:50:56.028+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingest_bronze_layer_dag', 'my_task', 'manual__2025-03-11T07:50:50+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/ingesting_pipeline.py', '--cfg-path', '/tmp/tmpz9s0ca3u']
[2025-03-11T07:50:56.030+0000] {standard_task_runner.py:85} INFO - Job 139: Subtask my_task
[2025-03-11T07:50:56.051+0000] {task_command.py:416} INFO - Running <TaskInstance: ingest_bronze_layer_dag.my_task manual__2025-03-11T07:50:50+00:00 [running]> on host 41ccd9d1769c
[2025-03-11T07:50:56.255+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='ingest_bronze_layer_dag' AIRFLOW_CTX_TASK_ID='my_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T07:50:50+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T07:50:50+00:00'
[2025-03-11T07:50:56.257+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:314: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-11T07:50:56.257+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingesting_pipeline.py", line 82, in my_task
    print(f"Task state: {ti.state}")
AttributeError: 'Query' object has no attribute 'state'
[2025-03-11T07:50:56.261+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=ingest_bronze_layer_dag, task_id=my_task, execution_date=20250311T075050, start_date=20250311T075056, end_date=20250311T075056
[2025-03-11T07:50:56.266+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 139 for task my_task ('Query' object has no attribute 'state'; 4427)
[2025-03-11T07:50:56.306+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-03-11T07:50:56.315+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
